portal/views/portal.py

```diff
@@ -1036,8 +1039,20 @@ def get_reporting_counts():
             counts['interventions'][interv.description] += 1
             if (any(doc.intervention == interv for doc in user.documents)):
                 counts['intervention_reports'][interv.description] += 1
-        for org in user.organizations:
-            counts['organizations'][org.name] += 1
+        if not user.organizations:
+            counts['organizations']['Unspecified'] += 1
```

Comment:
Gotcha.  Comments always help when such a situation isn't obvious - but now that I've read that, it is.